app/handler/provider_complaint.py

```diff
@@ -197,16 +197,16 @@ def report_complaint_to_user_in_forward_phase(
     send_email_with_rate_control(
         user,
         f"{ALERT_COMPLAINT_FORWARD_PHASE}_{origin.name()}",
-        user.email,
+        alias.mailbox.email,
```

Comment:
an alias can have several mailboxes so I think we might need to find a way to extract the right mailbox.

Reply:
How can we find what mailbox did it? I don't think the bounce message has any more info besides the alias

Reply:
just sent you an example of the complaint.